dataland-internal-storage/src/main/resources/application.properties

```diff
@@ -18,3 +18,14 @@ spring.mvc.throw-exception-if-no-handler-found=true
 spring.web.resources.add-mappings=false
 spring.jpa.open-in-view=false
 spring.datasource.hikari.maximum-pool-size=50
+dataland.backend.base-url=${INTERNAL_BACKEND_URL:http://backend:8080/api}
```

Comment:
That is funny: 
We default to `http://backend:8080/api` already in `docker_compose.yml`:`167`
Then, we default again to `http://backend:8080/api` right here, 
and to be very safe that we really default to `http://backend:8080/api`, we default again to `http://backend:8080/api` in `ConfigurationBackendControllerApi.kt`:`14`

Even though this is very safe - I think we can rely on that the defaulting-mechanism really works. In my opinion, chosing _only_ the application.properties file is a good choice.

Reply:
ok